vendor/assets/javascripts/initializers.js

```diff
@@ -64,4 +64,9 @@ $(document).on('turbolinks:load', function() {
         $(clone).first().removeClass('uk-hidden');
       });
   });
+  $('.js-featured-image-table-element').each(function() {
+    $(this).find('input').on('click', function(event) {
+      $('.js-featured-image').attr('src', event.target.parentElement.parentElement.dataset.url);
```

Comment:
The `parentElement.parentElement` feels too brittle. Can you add the `data` attribute on the TR itself and then just call `$this.data('imageUrl')`